src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -569,6 +596,10 @@ def test_conversion_as_lone_interface(self):
 
                 self.check_no_metadata_mutation()
 
+                self.check_run_conversion_default_backend_in_nwbconverter(
+                    nwbfile_path=self.nwbfile_path, backend="hdf5"
+                )
+
```

Comment:
I will mention briefly that in a follow-up, we should add the roundtrip (`check_read_nwb`) each time we create an NWB file over these 3 checks

I technically cheated and got away with it here by checking this first but for various interfaces it seems to cause some errors

Unrelated to the backend configuration, just related to the fact we've never systematically checked that all interfaces work out of the box when used in converters

Reply:
The long planned refactor of the tests is pending over us.